Makefile

```
@@ -145,14 +145,20 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Path used to import Gatekeeper manifests
-IMPORT_MANIFESTS_PATH ?= https://github.com/open-policy-agent/gatekeeper
+IMPORT_MANIFESTS_PATH ?= https://github.com/stolostron/gatekeeper
 TMP_IMPORT_MANIFESTS_PATH := $(shell mktemp -d)
 GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 
+GATEKEEPER_TAG = $(shell curl -L https://api.github.com/repos/stolostron/gatekeeper/tags | jq -r '.[].name' | sort --version-sort | grep $(shell echo $(GATEKEEPER_VERSION) | cut -d '.' -f 1-2) | tail -1 )
```

One more thing, sorry: I did some tests, and this needs to be a simply expanded variable, otherwise the HTTP request gets called 3 times during `import-manifests` (that's the `:=` assignment). I think we can also silence the `curl` command, since I don't find the output all that helpful since it's a quick turnaround:
```suggestion
GATEKEEPER_TAG := $(shell curl -sL https://api.github.com/repos/stolostron/gatekeeper/tags | jq -r '.[].name' | sort --version-sort | grep $(shell echo $(GATEKEEPER_VERSION) | cut -d '.' -f 1-2) | tail -1 )
```
ref: https://www.gnu.org/software/make/manual/html_node/Simple-Assignment.html

---

Great Great Thanks!!